packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -750,6 +750,14 @@ class AnimationController extends Animation<double>
     return result;
   }
 
+  /// Update the simulation without restarting the animation.
+  ///
+  /// The current simulation will be replaced with the provided [Simulation].
```

Comment:
This could mean that the animation is making a jump if the simulations are very different at the current point in time? Since there is no smooth transition from one simulation to the next one, we should probably document that here.